src/pages/Login.js

```diff
@@ -1,57 +1,62 @@
 import React from "react";
-import "../App.css";
-import "bootstrap/dist/css/bootstrap.min.css";
-import {Form,Button} from"react-bootstrap"
+import { Form, Button } from "react-bootstrap";
 import { useDispatch } from "react-redux";
-import {useHistory} from 'react-router-dom'
+import { useHistory } from "react-router-dom";
 
 export default function Login() {
-   let email=''
-   let password =''
-   const dispatch = useDispatch()
-   const history = useHistory()
+  let email = "";
+  let password = "";
+  const dispatch = useDispatch();
+  const history = useHistory();
 
-  const login = (e)=>{
+  const login = (e) => {
     e.preventDefault();
-    let user = {email:email,password:password}
-    dispatch({type:"LOGIN",payload:user})
-    history.goBack()
-    
-  }
+    let user = { email: email, password: password };
+    dispatch({ type: "LOGIN", payload: user });
+    history.goBack();
+  };
   return (
     <div className="App">
```

Comment:
this seems like a copy and paste. can you refactor into components that you're using? why are we building a new div named App here? This doesn't seem to make sense